src/lektorium/repo/local/storage.py

```diff
@@ -31,6 +32,13 @@
 run = functools.partial(subprocess.check_call, shell=True)
 
 
+def async_run(func, *args, **kwargs):
+    return get_event_loop().run_in_executor(
```

Comment:
Fine if it is working as expected. Could you please also create follow up to switch to asyncio way of running subprocesses?